install.sh

```diff
@@ -210,7 +210,16 @@ __install_bro() {
 			;;
 	esac
 	__install_packages bro broctl
-	chmod 2755 /opt/bro/logs
+	if [ -d /opt/bro/logs/ ]; then
+		chmod 2755 /opt/bro/logs
+	elif [ -d /var/log/bro/ ]; then
+		mkdir -p /opt/bro/logs/
+		chmod 2755 /opt/bro/logs
+		mv -f /var/log/bro /var/log/bro.orig
+		cd /var/log
+		ln -s /opt/bro/logs bro
```

Comment:
Could this just be `ln -s /opt/bro/logs /var/log/bro` without the `cd`ing? Or is there some nuance that I'm missing?

Reply:
I've had bad luck with that in the past.  I absolutely agree that it should do that, but sometimes it just doesn't make the link we both agree it should.  I've learned to not risk it, especially with a production shell script.